### PR TITLE
frontend/fix/#1736/frontEnd-bug-fixes

### PR DESCRIPTION
**Pull request by mohdshoaib18169**
## Type of Change

- [x ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Yatri Bug Fixes

bug Sheet -- https://github.com/nammayatri/nammayatri/issues/1594

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Comment by Rohit4997** [comment]
@mohdshoaib18169 Add a frontend label on PR

File: Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java [hunk 1]
```diff
@@ -3682,7 +3682,7 @@ public void drawRoute(final String json, final String style, final String trackC
                         {
                             List<LatLng> points = polylineOptions.getPoints();
                             LatLng source = points.get(points.size() - 1);
-                            upsertMarker(sourceMarker,String.valueOf(source.latitude),String.valueOf(source.longitude), 90, 0.5f, 0.5f);
+                            upsertMarker(sourceMarker,String.valueOf(source.latitude),String.valueOf(source.longitude), 160, 0.5f, 0.5f);
```
Comment (Vignesh-772) [inline Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java:3685]: try to pass this value from prescript in function call

**Review by Vignesh-772 — CHANGES_REQUESTED** [review]

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Components/DriverInfoCard/View.purs [hunk 2]
```diff
@@ -88,7 +88,7 @@ driverInfoViewSpecialZone push state =
               ][]
             , titleAndETA push state
             , otpAndWaitView push state
-            , separator (MarginHorizontal 16 16) (V 1) Color.grey900 (state.props.currentStage == RideStarted && (secondsToHms state.data.eta) /= "" && (state.props.isSpecialZone && (state.props.estimatedTime /= "--")))
+            , separator (MarginHorizontal 16 16) (V 1) Color.grey900 (state.props.currentStage == RideStarted && (secondsToHms state.data.eta) /= "" && (state.props.quotesOrEstimates == QUOTES && (state.props.estimatedTime /= "--")))
```
[the remark this answers is not in the document]
Comment (mohdshoaib18169) [reply Frontend/ui-customer/src/Components/DriverInfoCard/View.purs:91]: @ashishsingh101 , those are merged in mine , so no need to consider his changes

File: Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs [hunk 3]
```diff
@@ -57,6 +57,11 @@ getMerchant unit = case (decodeMerchantId (getMerchantId "")) of
   Just merchant -> merchant
   Nothing -> NAMMAYATRIPARTNER
 
+getMerchantVehicleSize :: Unit -> Int
+getMerchantVehicleSize unit = 
+  case getMerchant unit of 
+    _ -> 90
+
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs:64]: refactor this function

Comment (mohdshoaib18169) [reply Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs:64]: @ashishsingh101 , this function is for future reference , like in future if driver vehicle size needs to be change , 
as it changed in Yatri app

Comment (ashishsingh101) [reply Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs]: then you can pass constant value from purescript and later if required will create a function for this

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs (hunk 3 above)
Comment (ashishsingh101) [reply Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs]: ok fine

File: Frontend/ui-common/src/Engineering/Helpers/JBridge.purs [hunk 4]
```diff
@@ -325,6 +325,7 @@ type Locations = {
 type SpecialLocationTag = {
     sourceSpecialTagIcon :: String
   , destSpecialTagIcon :: String
+  , vehicleSize :: Int
```
[the remark this answers is not in the document]
Comment (ashishsingh101) [reply Frontend/ui-common/src/Engineering/Helpers/JBridge.purs]: @mohdshoaib18169 I was talking about type `SpecialLocationTag` you can rename it

**Review by Vignesh-772 — APPROVED** [review]

File: Frontend/ui-common/src/Engineering/Helpers/JBridge.purs (hunk 4 above)
[the remark this answers is not in the document]
Comment (mohdshoaib18169) [reply Frontend/ui-common/src/Engineering/Helpers/JBridge.purs]: renamed

**Review by MercyQueen — APPROVED** [review]

**Pull request merged**